### PR TITLE
Fix LBRY playback.

### PR DESCRIPTION
**Pull request by FireMasterK**

### CODE REVIEW

File: src/components/VideoPlayer.vue [hunk 1]
```diff
@@ -273,7 +273,7 @@ export default {
 
             const lbry = this.getPreferenceBoolean("disableLBRY", false)
                 ? null
-                : this.video.videoStreams.filter(stream => stream.quality === "LBRY")[0];
+                : this.video.videoStreams.filter(stream => stream.quality.indexOf("LBRY") !== -1)[0];
```
Comment (Bnyro) [inline src/components/VideoPlayer.vue:276]: Just a question, why not using `stream.quality.includes("LBRY")` instead?

Comment (FireMasterK) [reply src/components/VideoPlayer.vue:276]: Didn't know such a thing existed lol, I've always looked for a "contains" method, thanks!

**Comment by Bnyro** [comment]
Is there anything still blocking that PR?

**Comment by FireMasterK** [comment]
The problem is it doesn't actually 'fix' anything, we still get a 401 unfortunately.

I'll send odysee an email to see what happens.

**Comment by Bnyro** [comment]
Are you sure? I was pretty sure it worked fine when I tested it with LibreTube :thinking:

**Pull request closed without merging**